pkg/internal/loader/kubernetes/components_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 
 	"github.com/dapr/dapr/pkg/apis/components/v1alpha1"
-	subscriptions "github.com/dapr/dapr/pkg/apis/subscriptions/v1alpha1"
+	subapi "github.com/dapr/dapr/pkg/apis/subscriptions/v2alpha1"
```

Comment:
is this right? You are now using the v2alpha import? should that `v2` be included in the import name: `subapi` -> `subv2api`

Reply:
Yes, that's right- In Kubernetes (and internally really), `v2alpha1` is our "target" and stored version where created `v1alpha1` Subscription resources are converted into `v2alpha1` types during admission. Dapr Kube clients will only deal with the `v2alpha1` type. 

This `ListSubscriptionsV2` test func was incorrectly using `v1alpha`, event though the func name itself is `V2` (!)